src/components/DisplayNames/types.ts

```diff
@@ -20,6 +20,9 @@ type DisplayNamesProps = ForwardedFSClassProps & {
     /** The full title of the DisplayNames component (not split up) */
     fullTitle: string;
 
+    /** If the full title needs to be parsed */
```

Comment:
Can you update this comment to make clearer when you'd use this? Perhaps with an example.